frontend/src/views/assignments/offer-actions.js

```diff
@@ -56,14 +64,24 @@ function OfferActionButtons(props) {
 
     return (
         <React.Fragment>
-            <ButtonGroup>
-                <Button onClick={createOffers}>Create Offer</Button>
-                <Button onClick={withdrawOffers}>Withdraw Offer</Button>
-                <Button onClick={emailOffers}>Email Offer</Button>
-                <Button onClick={nagOffers}>Nag Offer</Button>
-                <Button onClick={acceptOffers}>Set as Accepted</Button>
-                <Button onClick={rejectOffers}>Set as Rejected</Button>
-            </ButtonGroup>
+            <ActionButton icon={<FaPlusCircle />} onClick={createOffers}>
+                Create Offer
+            </ActionButton>
+            <ActionButton icon={<FaMinusCircle />} onClick={withdrawOffers}>
+                Withdraw Offer
+            </ActionButton>
+            <ActionButton icon={<FaArrowCircleRight />} onClick={emailOffers}>
```

Comment:
We can do better with the icons. Search through and try to think of what icons would be meaningful (e.g, here a "mail" icon would be better)

Reply:
I really looked. If you would like to suggest one, I would appreciate it, because I scrolled through the react icons/fa icons for about 1hr trying to find a set of icons that we can use for mail. The only one that doesn't really make super intuitive sense is the email one, I think.

Reply:
`<i class="fa fa-envelope"></i>` for mail
`<i class="fa fa-user-times"></i>` or `<i class="fa fa-ban"></i>` for withdraw 
`<i class="fa fa-ban"></i>` for nag
`<i class="fa fa-check-circle"></i>` or `<i class="fa fa-pencil-square-o"></i>` for accept

Something like that

Reply:
`FaUserPlus` for add offer
`FaUserTimes` for withdraw offer
`FaEnvelope` for Email
`FaUserClock` for nag (am i misunderstanding what this does? it seems like ban doesn't really make sense...)
`FaCheck` for Accept Offer
`FaBan` for Reject Offer

Reply:
A nag is like a reprimand.